sources/packages/backend/libs/utilities/src/system-configurations-constants.ts

```diff
@@ -57,3 +57,8 @@ export const ORM_CACHE_REDIS_RETRY_INTERVAL = 60 * 1000;
 export const COE_WINDOW = 21;
 
 export const COE_DENIED_REASON_OTHER_ID = 1;
+
+/**
+ * SFTP directory name used to archive files.
+ */
+export const SFTP_ARCHIVE_DIRECTORY = "Archive";
```

Comment:
We can use the system configuration constants in integrations. 

https://github.com/bcgov/SIMS/blob/main/sources/packages/backend/libs/integrations/src/constants/system-configurations-constants.ts